packages/@sparrow-teams/src/features/team-explorer/layout/TeamExplorer.svelte

```diff
@@ -234,7 +234,11 @@
             <span
               class="ms-3 my-auto ellipsis overflow-hidden heading"
               style="font-size: 28px;"
-              >{openTeam?.name || ""}
+              >{#if isGuestUser}
```

Comment:
We don't need this if else condition, when we create dummy team for guest user we already that with "Team" name so no  need for this condition.

Reply:
I will fix it.